sound/soc/intel/boards/Kconfig

```diff
@@ -536,6 +536,30 @@ config SND_SOC_INTEL_TGL_RT711_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE && SND_SOC_SOF_INTEL_SOUNDWIRE
 
+if SND_SOC_SOF_ELKHARTLAKE
+
+config SND_SOC_INTEL_EHL_RT5660_MACH
+	tristate "EHL with RT5660 in I2S mode"
+	depends on I2C && ACPI
+	depends on MFD_INTEL_LPSS || COMPILE_TEST
+	select SND_SOC_RT5660
+	help
+	  This adds support for ASoC machine driver for Elkhart Lake platform
+	  with RT5660 I2S audio codec.
+	  Say Y or m if you have such a device. This is a recommended option.
+	  If unsure select "N".
```

Comment:
I know this is a 1000-times repeated copy-paste, but thinking about it... "recommended Y or m" - ok. "if unsure select N" - emmm... wait. So, we recommend Y or m. But if you're unsure, we recommend N. So, we recommend Y or m only if you're sure. But if you're sure you don't need our recommendation :-D m?

Reply:
in practice the choice is restricted by the if on top. You will only be able to choose M  or N if SND_SOC_SOF_ELKHARTLAKE is compiled as M.

Reply:
@lyakh @plbossart what about change to "Say Y or M if you have such a device. If unsure select "N"."

Reply:
@libinyang that would sound logical at least, yes. But I'm in fact in general not a huge fan of these statements. They are stating the obvious to anyone, who is configuring a kernel build, aren't they? Of course that person understands, that this option enables support for a specific device, and that y, m, and n are possible - in general. The default is "N" right? If anything I'd just reinforce that because that might not always be obvious. So, "If unsure select 'N'" at least carries some information. Or we could leave that whole paragraph out completely.

Reply:
@lyakh OK, I will remove the paragraph.